Concentration/Card.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Card.swift
+//  Concentration
+//
+//  Created by Oren Dinur on 08/02/2022.
+//
+
+import Foundation
+
+struct Card
+{
+    var isFaceUp = false
+    var isMatched = false
+    var identifier: Int
+    
+    static var identifierFactory = 0
+    
+    static func getUniqueIdentifier() -> Int {
+        identifierFactory += 1
+        return identifierFactory
+    }
+    
+    init(){
```

Comment:
Please follow this style:
```suggestion
    init() {
```